roles/pulp3-devel/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- block:
+
+  - name: Install several useful packages
+    package:
+      name:
+        - bash-completion
+        - dnf-utils
+        - dstat
+        - fd-find
+        - fedora-easy-karma
+        - fpaste
+        - fzf
+        - git
+        - htop
+        - httpie
+        - iotop
+        - jnettop
+        - jq
+        - ncdu
+        - python-django-bash-completion
+        - python-ipdb
```

Comment:
I use this all the time, but it is not in our docs. For packages in this category, IMO, devs should install on their own. Its really easy to make a personal role and add it to a Vagrantfile or whatever. This way, we don't waste time waiting on the installation of tools that other people sometimes use.

Reply:
On second thought, this should be removed. If we install this, it should be pip installed, not dnf.